src/hooks/useFetchPopkeapi.ts

```diff
@@ -1,6 +1,9 @@
 import { useEffect, useState } from "react"
 
 
+
+const localCache: { [key: string]: any } = {};
+
 interface FetchData<T>{
```

Comment:
Same thing with the generics here, if this hook is only used in one place, just move the types here and remove the generic